### DISCUSSION TITLE
Will this work with 1.064

### DISCUSSION THREAD

**Discussion by roopesh** [jlvaillant/intellicenter#40] [General]
I just got a notification to update to 1.064.  Is anyone running 1.064 with this custom component?

**Comment by TheRydad** [comment]
I just installed this integration today and was already on 1.064 when I did. It discovered just fine and the sensors and work and I successfully tested turning a light on and off. The problem is that it will only work for 20 seconds and then all entities go "unavailable" for 30 seconds. This repeats over and over and always exactly at 20/30 second intervals.

I have reloaded the component and rebooted then Intellicenter panel twice. I do have my HA and my Intellicenter on separate VLANs, so I might try changing them to the same VLAN to see if it makes a difference.

Please post if you get it working (or not).

**Comment by roopesh** [comment]
I'm not going to upgrade until I see this updated for 1.064 😃 .  I rely more on local than cloud.

**Reply by TheRydad** [reply]
I had another "supported" integration that I had noticed went offline when I updated HA to 2022.2. I restored back to 2022.1 and it fixed that integration, but this Intellicenter one still has the same problem i described in the other thread. I might see if I can revert back to 1.047. 

According to the notices I keep getting emailed, 1.047 will stop working for remote access on March 22. I rely a lot on the Pentair app to control the pool remotely, though, so I am reluctant to mess with it too much. My guess is Pentair changed the API which is why the 1.064 update also requires a completely new app (at least on iOS).

**Comment by jlvaillant** [comment]
When I checked this morning , I was running 1.064.
I noticed the iOS IntelliCenter was kicking me out after a few seconds claiming Service mode was on. 
Went back to the panel and the **Service** mode was set to **Timeout** (no idea what that means). By changing it to **Auto** my connection became stable again. So I guess I would check that first.
Regarding stability with HA 2022.x I haven't had a chance to look at it but will get to it ASAP.

**Reply by TheRydad** [reply]
Thanks for the reply. I'll check that when I get home in a few hours.

**Reply by TheRydad** [reply]
My panel was in "Auto" already. Still exhibiting the same behavior.

Do you know if the communication protocol relies on any mDNS or Bonjour traffic? My HA is connected to a switch in my house and the Intellicenter is connected to a switch in a back house. Everything is hard wired and trunked, but it is UniFi equipment which is known to have some glitchy switch to switch broadcast/mDNS/Bonjour implementations. Both are on the same VLAN.

FWIW, "Timeout" is for times when someone is doing service on the pool and it needs to stay off for a certain amount of time. They can set the timeout to something like 3 hours and leave. 3 hours later, the pool will go back to its normal schedule. My guess is that someone set a timeout on your pool at some point in the past and it expired and the pool went back to normal operation.

"Service" locks out all schedules and remote operations indefinitely. Someone cleaning the pool, for example, would put it in service mode so they can locally control the pumps, etc. to run suction vacuums, empty skimmer baskets, etc. When they are done they would put it back to "Auto" so all schedules run and remote operations work.

**Reply by roopesh** [reply]
So @jlvaillant  you're not seeing the intermittent connectivity errors on 1.064?  I'm willing to wait to update, since I rely more on the HA integration than the app.

**Comment by sellison2** [comment]
The issue with everything going available constantly is fixed in v1.1.0

**Comment by smartqasa** [comment]
Just installed the integration. Intellicenter 1.064 with HA Core 2022.4.2 and Supervisor 2022.4.0. 

Two observations:
1. Circuit groups not showing up as HA entities. 
2. Hybrid heater (for UltraTemp ETi Hybrid Heater) controls not visible on HA.
